e2e/jest-puppeteer.config.js

```diff
@@ -16,7 +16,7 @@ const NEW_CHROME_SWITCHES = [
   '--no-zygote',
   '--no-sandbox', // required for --no-zygote flag
   '--safebrowsing-disable-auto-update',
-  '--window-size=1280,1024'
+  '--window-size=1300,1024'
```

Comment:
Just a bit wider to prevent horizontal scrollbar appearing in browser.
<img width="492" alt="Screen Shot 2021-01-27 at 1 38 43 PM" src="https://user-images.githubusercontent.com/35533885/106044400-af349680-60ad-11eb-803b-dd0c8130903e.png">